cmd/main.go

```diff
@@ -160,6 +164,11 @@ func validateFormat(stdout string, reportPath []string) {
 }
 
 func preRun(cmd *cobra.Command, args []string) {
+	shouldIgnore, _ := cmd.Flags().GetString("ignore-on-exit")
```

Comment:
Even if you copied the code from Kics, please follow our code conventions.

You don't need to put the string literal, because you declared this string above.

And you don't even need to use `GetString` at all, because the value will already be in the `ignoreOnExitVar`.